packages/core/database/lib/entity-manager/relations/cloning/regular-relations.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { cleanInverseOrderColumn } = require('../../regular-relations');
+
+const replaceRegularRelations = async ({ id, cloneId, attribute, transaction: trx }) => {
+  const { joinTable } = attribute;
+  const { joinColumn, inverseJoinColumn } = joinTable;
+
+  // We are effectively stealing the relation from the cloned entity
+  await strapi.db.entityManager
+    .createQueryBuilder(joinTable.name)
+    .update({ [joinColumn.name]: id })
+    .where({ [joinColumn.name]: cloneId })
+    // TODO: Exclude some relations from being replaced
+    // .where({ $not: { [inverseJoinColumn.name]: relationsToDeleteIds } })
+    .onConflict([joinColumn.name, inverseJoinColumn.name])
+    .ignore()
+    .transacting(trx)
+    .execute();
+};
+
+const cloneRegularRelations = async ({ id, cloneId, attribute, transaction: trx }) => {
+  const { joinTable } = attribute;
+  const { joinColumn, inverseJoinColumn, orderColumnName, inverseOrderColumnName } = joinTable;
+  const connection = strapi.db.getConnection();
+
+  // Get the columns to select
+  const columns = [joinColumn.name, inverseJoinColumn.name];
+  if (orderColumnName) columns.push(orderColumnName);
+  if (inverseOrderColumnName) columns.push(inverseOrderColumnName);
+  if (joinTable.on) columns.push(...Object.keys(joinTable.on));
+
+  const selectStatement = connection
+    .select(
+      // Override joinColumn with the new id
+      { [joinColumn.name]: id },
+      // The rest of columns will be the same
+      ...columns.slice(1)
+    )
+    .where(joinColumn.name, cloneId)
+    // TODO: Exclude some relations from being replaced
+    // .where({ $not: { [inverseJoinColumn.name]: relationsToDeleteIds } })
+    .from(joinTable.name)
+    .toSQL();
```

Comment:
```suggestion
  const selectStatement = connection
    .select(
      // Override joinColumn with the new id
      { [joinColumn.name]: cloneId },
      // The rest of columns will be the same
      ...columns.slice(1)
    )
    .where(joinColumn.name, id)
    // TODO: Exclude some relations from being replaced
    // .where({ $not: { [inverseJoinColumn.name]: relationsToDeleteIds } })
    .from(joinTable.name)
    .toSQL();
```

Are these ids backwards?

Reply:
I guess it depends what we consider cloneId 🤔 , I consider:
-  `cloneId` the id of the original entity
- `id` the id of the new entity 

By following that definition the original sql is correct. 
Do you think that is clear? (or any variable name alternative)

Reply:
completely 🤔 I will come back to this with more relevant variable names.

Reply:
@jhoward1994 I have renamed those two variables to `targetId` and `sourceId`  you can see the change in this commit [`bf66b84` (#16062)](https://github.com/strapi/strapi/pull/16062/commits/bf66b845168098de014649e40bb30ce67e7bd1b9)

wdyt?